conode/run_conode.sh

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env bash
```

Comment:
> You'd have to be able to import files after the compilation

I was merely talking about the bash script. Even a simple  `git clone gopkg.in/dedis/onet.v1/app/run_conode.sh && ./run_conode.sh $@` would be better no ?